mslib/mscolab/server.py

```diff
@@ -482,12 +495,16 @@ def update_operation():
 def get_operation_details():
     op_id = request.args.get('op_id', request.form.get('op_id', None))
     user = g.user
+    result = str(fm.get_operation_details(int(op_id), user))
+    if result is False:
+        return "False"
     return json.dumps(fm.get_operation_details(int(op_id), user))
```

Comment:
get_operation_details is called twice here in the "normal" case.

Reply:
one too much, good catch